workshops/views.py

```diff
@@ -1061,16 +1062,19 @@ def event_import(request):
 
         if 'slug' not in tags:
             # there are no HTML tags, so let's try the old method
-            url = generate_url_to_event_index(url)
+            index_url = generate_url_to_event_index(url)
 
             # fetch page
-            response = requests.get(url)
+            response = requests.get(index_url)
```

Comment:
Not something you're changing in this PR, but I just noticed that `generate_url_to_event_index` will return `None` instead of raising a `ValueError` or something if the passed URL doesn't match the expected regexp.  We probably want to make the function raise an exception when it can't find the raw source, and then catch and handle that exception in here.